cmd/preflight/cmd/check_operator.go

```diff
@@ -71,7 +73,7 @@ func ensureIndexImageConfigIsSet() error {
 }
 
 // checkOperatorRunE executes checkOperator using the user args to inform the execution.
-func checkOperatorRunE(cmd *cobra.Command, args []string) error {
```

Comment:
I think I'd like to see this, and potentially the equivalent for container checks, change function signatures.

By convention, checkOperatorRunE's signature matches the RunE signature definition. We're changing this to accept dependencies in the function signature, which to me, strays from the original intention behind the convention's use.

I'm thinking these should be `newCheckOperatorRunE` and return a RunE signature.

Reply:
The order got messed up in my refactor I can fix that, do you still feel they still need a new name?

Reply:
It's a small detail; I'd say it can't hurt. Your call.

Reply:
Why return a RunE signature? I get renaming them (not a RunE technically anymore) but I don't understand the return part.

Reply:
It would let you define `RunE: NewRunEFunctionWithThese(deps deps)`. Either that, or you use the anon function. They accomplish the same thing.

Reply:
@komish You can't do what you are suggesting, if adding more args it has to be in a closure, meaning you can't call with params.

Reply:
@acornett21 I see what he's getting at. If NewRunEFunctionWithThese() has a return type of `func (cmd *cobra.Command, args []string) error` then you can call it as he states. 

However, that would require re-working the function we have to account for that. IMHO how we have it is cleaner. As @komish said, it accomplishes the same thing.